### PR TITLE
chore: publish release branch to latest

### PR DESCRIPTION
**Pull request by erinleigh90**
Point release publish to latest tag

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ashwinkumar6 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
 		"unlink-all": "lerna exec --no-bail --parallel -- yarn unlink; exit 0",
 		"publish:preid": "./scripts/preid-env-vars-exist.sh && lerna publish --canary --force-publish --dist-tag=${PREID_PREFIX} --preid=${PREID_PREFIX}${PREID_HASH_SUFFIX} --yes",
 		"publish:main": "lerna publish --canary --force-publish --dist-tag=unstable --preid=unstable${PREID_HASH_SUFFIX} --yes",
-		"publish:release": "lerna publish --conventional-commits --dist-tag=v6 --message 'chore(release): Publish [ci skip]' --yes",
+		"publish:release": "lerna publish --conventional-commits --dist-tag=latest --message 'chore(release): Publish [ci skip]' --yes",
```
Comment (elorzafe) [inline package.json:32]: I think we can remove `--dist-tag` flag because by default will point to `latest`

Comment (elorzafe) [reply package.json:32]: in the past it was like that

Comment (erinleigh90) [reply package.json:32]: Ah got it, let me adjust

**Review by elorzafe — APPROVED** [review]
Nice

Thanks @erinleigh90 !

**Review by ashwinkumar6 — APPROVED** [review]

**Review by kvramyasri7 — APPROVED** [review]

**Review by jimblanc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**